andela/denethor-ah-backend
# #163478797 add GET and DELETE routes for users

#### What does this PR do?
 This pull request implements GET and DELETE route for users.

#### Description of Task to be completed?
- Add GET route to fetch all authors
- Add GET route for details of a particular user
- Add DELETE route for users

#### How should this be manually tested?
- Clone the repo
- Checkout branch `feature/163478797/social-login`
- Run `npm i` to install dependencies
- Set up `.env` file as specified in the `.env.sample file`
- Start server with `npm run start:dev`
- Check out routes with the Postman app.

#### What are the relevant pivotal tracker stories?
[#163478808](https://www.pivotaltracker.com/story/show/163478808)
